### DISCUSSION TITLE
Common Offset Reporting

### DISCUSSION THREAD

**Discussion by KnugiHK** [KnugiHK/WhatsApp-Chat-Exporter#47] [General]
The offsets predefined in the exporter did not work for you. Luckily it can be brute-force, and you are led to here. Feel free to report the offsets for IV and the database as shown in the output.

Example:
IV: 67
DB: 191

**Comment by silasjelley** [comment]
> The offsets of your IV and database are 67 and 196, respectively.

**Reply by KnugiHK** [reply]
Thank you very much for the report! Offset added.

**Reply by Whoneon** [reply]
Hi. I have the same offset, yet the program had to Bruteforce to find those. I made sure to run pip -U on all packages required

**Reply by KnugiHK** [reply]
Did you install via PyPi? The offset is added but not yet release to PyPi.
